tic-tac-toe/src/components/control-pannel/control-pannel.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
+import './control-pannel.css';
+
+const GameModePannel = ({activeGameMode, onChangeMode, outerClassName}) => {
+    const innerClassName = 'mode-panel';
+    const className = `${outerClassName} ${innerClassName}`
+    const buttonsInfo = [
+        {name: 'human', label: 'human'},
+        {name: 'robot', label: 'robot'},
+    ];
```

Comment:
that is constant so please keep it in global scope of component

Reply:
I made it.
```suggestion
import React from 'react';
import classes from  './control-pannel.module.css';

const buttonsInfo = [
  {name: 'human', label: 'human'},
  {name: 'robot', label: 'robot'},
];

const GameModePannel = ({activeGameMode, onChangeMode}) => {
```